standard/lpdb/count.lua

```diff
@@ -0,0 +1,236 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Count
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+local Team = require('Module:Team')
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local Count = {}
+
+
+---@deprecated `lpdb_game` is deprecated
+---Counts the amount of games played on a wiki
+---@param args table
+---@return integer?
+function Count.games(args)
```

Comment:
fyi: keep in mind that for new wikis that do not have legacy storage this will not work (always return 0)